examples/api/lib/cupertino/sheet/cupertino_sheet.0.dart

```diff
@@ -0,0 +1,85 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+
+/// Flutter code sample for [CupertinoSheetRoute].
+
+void main() {
+  runApp(const CupertinoSheetApp());
+}
+
+class CupertinoSheetApp extends StatelessWidget {
+  const CupertinoSheetApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const CupertinoApp(
+      title: 'Cupertino Sheet',
+      home: HomePage(),
+    );
+  }
+}
+
+class HomePage extends StatelessWidget {
+  const HomePage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoPageScaffold(
+      navigationBar: const CupertinoNavigationBar(
+        middle: Text('Sheet Example'),
+        automaticBackgroundVisibility: false,
+      ),
+      child: Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            CupertinoButton.filled(
+              onPressed: () {
+                Navigator.of(context).push(CupertinoSheetRoute<void>(
+                  builder: (BuildContext context) => const SheetScaffold()
```

Comment:
Super nit: Make SheetScaffold private so it's clear that this is something defined in the example and not in the framework?

Here and some others below.